### PR TITLE
chore: rug portfolio websocket

### PR DESCRIPTION
**Pull request by dereknelson**

### CODE REVIEW

File: src/screens/WalletScreen/index.tsx [hunk 1]
```diff
@@ -64,7 +62,6 @@ const WalletScreen: React.FC<any> = ({ navigation, route }) => {
   const { userAccounts } = useUserAccounts();
```
Comment (benisgold) [inline src/screens/WalletScreen/index.tsx:62]: there's some leftover code such as this line that is now unused

File: src/redux/explorer.ts [hunk 2]
```diff
@@ -439,23 +439,6 @@ export const explorerInit = () => async (
...
- * @param currency The currency to use.
- */
-export const emitPortfolioRequest = (address: string, currency?: string) => (
-  _: Dispatch,
-  getState: AppGetState
-) => {
-  const nativeCurrency = currency || getState().settings.nativeCurrency;
-  const { addressSocket } = getState().explorer;
-
-  addressSocket?.emit(...portfolioSubscription(address, nativeCurrency));
```
Comment (benisgold) [inline src/redux/explorer.ts:456]: u can remove `portfolioSubscription`

File: src/hooks/usePortfolios.ts [hunk 3]
```diff
@@ -1,50 +0,0 @@
...
-          total[key as keyof typeof total] += portfolios[address][
-            key as keyof typeof portfolios[typeof address]
-          ]!;
-        }
-      });
-    });
-    logger.log('💰 wallet totals', JSON.stringify(total, null, 2));
-    keys(total).forEach(key => {
-      const data = { [key]: total[key as keyof typeof total] };
-      analyticsV2.identify(data);
```
Comment (benisgold) [inline src/hooks/usePortfolios.ts:42]: might not be relevant for this PR, but do these analytics need to be migrated elsewhere?

File: src/redux/explorer.ts (hunk 2 above)
Comment (benisgold) [reply src/redux/explorer.ts:456]: actually seems like there might be some stuff to be removed in `listenOnAddressMessages` as well

File: src/hooks/usePortfolios.ts (hunk 3 above)
Comment (dereknelson) [reply src/hooks/usePortfolios.ts:42]: no they're mostly from backend now

**Pull request merged**